docs/machine-learning/tutorials/image-classification.md

```diff
@@ -111,336 +111,249 @@ toaster2.png	appliance
 
-| Class         | Probability   |
-| ------------- | ------------- |
-| `Food`        |  0.001        |
-| `Toy`         |  0.95         |
-| `Appliance`   |  0.06         |
+An ML.NET model pipeline is a chain of estimators. Note that no execution happens during pipeline construction. The estimator objects are created but not executed.
 
-Append the `TensorFlowTransform` to the `estimator` with the following line of code:
+1. Add the estimators to load, resize and extract the pixels from the image data:
 
-[!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
+    [!code-csharp[ImageTransforms](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ImageTransforms)]
 
-### Choose a training algorithm
+    The image data needs to be processed into the format that the TensorFlow model expects. In this case, the images are loaded into memory, resized to a consistent size, and the pixels are extracted into a numeric vector.
 
-To add the training algorithm, call the `mlContext.MulticlassClassification.Trainers.LbfgsMaximumEntropy()` wrapper method.  The [LbfgsMaximumEntropy](xref:Microsoft.ML.Trainers.LbfgsMaximumEntropyMulticlassTrainer) is appended to the `estimator` and accepts the Inception image features (`softmax2_pre_activation`) and the `Label` input parameters to learn from the historic data.  Add the trainer with the following code:
+1. Add the estimator to load the TensorFlow model, and score it:
 
-[!code-csharp[AddTrainer](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddTrainer)]
+    [!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
 
-You also need to map the `predictedlabel` to the `predictedlabelvalue`:
+    The next stage in the pipeline is to load the TensorFlow model into memory, then process the vector of pixel values through the TensorFlow model network. Applying inputs to a deep learning model is referred to as **Scoring**. When using the model in its entirety, scoring makes an inference, or prediction. 
 
-[!code-csharp[MapValueToKey2](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey2)]
+    In this case, you use all of the TensorFlow model except the last layer, which is the layer that makes the inference. The output of the penultimate layer is labeled `softmax_2_preactivation`. This output of this layer is effectively a vector of features that characterize the original input images.
 
-The `Fit()` method trains your model by transforming the dataset and applying the training. Fit the model to the training dataset and return the trained model by adding the following as the next line of code in the `ReuseAndTuneInceptionModel()` method:
+    This feature vector generated by the TensorFlow model will be used as input to the ML.NET trainer below.
```

Comment:
Suggestion: The feature vector generated by the TensorFlow model is used as input to an ML.NET trainer.

Question: Is it a trainer or estimator? Especially since it's the value-to-key mapping and not the algorithm.

Reply:
I hedged and changed it to ML.NET training algorithm